### ISSUE TITLE
FIt-SNE wrapper requires upgrading to 1.2.0

### ISSUE THREAD

**Issue by linqiaozhi** [satijalab/seurat#2792]
We released Version 1.2.0 of FIt-SNE, which means that Seurat's wrapper will also need updating to use the latest executable. In the meantime, users should download the Version 1.1.0, as mentioned in the Seurat documentation. 

I should be able to update the [wrapper](https://github.com/satijalab/seurat/blob/87e2454817ed1d5d5aa2e9c949b9231f2231802f/R/dimensional_reduction.R#L1682-L1872) soon, but I wanted to submit this issue in case 1) someone encounters this error in the meantime or 2) someone else ends up updating it before I do.

**Comment by BenjaminDoran** [comment]
Is this just matter of changing the grep statement or do the new defaults for fast_tsne 1.2 also need to be taken into account?

**Comment by linqiaozhi** [comment]
@BenjaminDoran Good question. I think that the new defaults should be implemented too. The reason being that if someone reports they used a specific version of FIt-SNE (default values), for the purposes of reproducibility that should mean the same thing whether they used Seurat or not.

Also, we released 1.2.1 today, so should upgrade to that. There was a bug preventing 1.2.0 from working on Windows, which is now resolved with the new release.
